archive/xar.ksy

```diff
@@ -0,0 +1,120 @@
+meta:
+  id: xar
+  title: XAR (eXtensible ARchive)
+  file-extension:
+    - pkg
+    - xar
+    - xip
```

Comment:
By convention, `file-extension` entries should be listed in the order from the most typical/popular extension to the least common one (https://doc.kaitai.io/ksy_style_guide.html#meta):

> `file-extension` — if there’s only one extension, MUST be a string; if there are several, MUST be an sequence in block form and SHOULD order extensions from most popular extension to least popular.

I wonder if it makes sense to put `pkg` above `xar` here?

Reply:
because `pkg` is used for many other formats as well I am not so sure about that. I have no idea which extension is more popular these days. I would say that `xar` is at least the most descriptive.